.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```

Comment:
Do not import unrelated files to project, beside, I strongly recommand VS Code instead of VS.